packages/govern-core/contracts/lib/MiniACL.sol

```diff
@@ -53,6 +57,7 @@ contract MiniACL {
         _revoke(_role, _who);
     }
 
+    // Note: I'm a bit hesitant about this, if the root is trustless enough we should be able to get rid of this
```

Comment:
that's the thing, we want to leave the door open to non-trustless roots that have limited powers enforced by the acl